### DISCUSSION TITLE
Improve Auth Web empty state

### DISCUSSION THREAD

**Discussion by entibo** [ente-io/ente#4134] [Enhancements]
After creating an account for Ente Auth, the web version at https://auth.ente.io/auth is empty except for a button stating "Download our mobile app to manage your secrets". This made me think that I couldn't access my codes on the web app.
I suggest adding a placeholder message indicating that "your codes will appear here".

Other discussion (not by me):
https://old.reddit.com/r/enteio/comments/1gwdc99/is_there_a_web_app_for_ente_auth/

**Comment by mnvr** [comment]
Have made this change. Thank you.

<img width="444" alt="Screenshot 2024-11-22 at 08 59 27" src="https://github.com/user-attachments/assets/56eb7040-b1fa-4f0a-a501-1242e961d402">
